packages/ember-auto-import/ts/webpack.ts

```diff
@@ -32,8 +32,8 @@ const EXTENSIONS = ['.js', '.ts', '.json'];
 
 const debug = makeDebug('ember-auto-import:webpack');
 
-registerHelper('js-string-escape', jsStringEscape);
-registerHelper('join', function (list, connector) {
+Handlebars.registerHelper('js-string-escape', jsStringEscape);
+Handlebars.registerHelper('join', function (list, connector) {
```

Comment:
The API docs only show that kind of usage, and `registerHelper` accesses `this.`. Not sure why it worked before, but as soon as the compilation output from a new TS versions changed this be like `(0, handlebars_1.registerHelper)('join', ...)`, it would lose access to `this`.